### ISSUE TITLE
Rename reporting_activities endpoint

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#2278]
###Description of the Tech Debt###

ReportingActivity data model was renamed to Activity but the endpoint remained reporting_activities (to minimize the impact of renaming). it would be good to rename the endpoint to reflect the data model properly. (this includes updating FE calls and BE tests)

###Tech Debt Triage#
The purpose of our technical debt triage process is to analyze technical debt to determine risk level of the technical debt and the value in tackling that technical debt.

Risk Value Scoring:

| Level  | Value |
| ------ | ----- |
| High   | 3     |
| Medium | 2     |
| Low    | 1     |

| Technical Debt - Risk Types                                                                                                   | Level | Value |
| ----------------------------------------------------------------------------------------------------------------------------- | ----- | ----- |
| Business Area Risk - Risk of business area visibility / damage to user experience                                             |       |       |
| Developer Fault Risk - How likely will this tech debt cause a future error related to coding on top of it                     |       |       |
| System Fault Risk - Risk of system errors or application downtime                                                             |       |       |
| Time Scale Risk - Compound risk effect if left alone. How much more difficult to fix or dangerous will this become over time? |       |       |
| Time Sink Risk - How much will this tech debt slow the development process down                                               |       |       |
| TOTAL SCORE:                                                                                                                  |       |       |

**Comment by patriciarussellCAS** [comment]
@BCerki or @Sepehr-Sobhani is this ticket still needed? 
Tagging @dleard for visibility.

**Comment by Sepehr-Sobhani** [comment]
Yes, we need to update the `"/reporting_activities"` endpoint. I noticed that we have a duplicate endpoint named `"/activities"`, but it doesn’t use the response schema or cache the response. To avoid confusion and take advantage of caching and a consistent response schema, it would be better to consolidate into a single endpoint.
